include/smtlib_reader.h

```diff
@@ -150,6 +150,17 @@ class SmtLibReader
    */
   virtual void pop(uint64_t num = 1);
 
+  /** Mark a term with an attribute
+   *  This currently is unimplemented and just prints a warning.
```

Comment:
Unimplemented? Or is it implemented but the attribute is ignored?

Reply:
Good point, fixed in 793be8ca80bf09cf834920d55f3564cdc93cdb64.